### DISCUSSION TITLE
Docker followTag

### DISCUSSION THREAD

**Discussion by SimonAdameit** [renovatebot/renovate#15223] [Request Help] [answered]
Hello,

I'd like renovate to follow a docker tag and only update the digest. The tag does not represent a version. An image name would for example be: `ghcr.io/myorg/myimage:prod@sha256...`.

How can I configure renovate to follow e.g. the`prod` tag for this image and update the digest when a image is pushed with the tag?

Cheers,
Simon

**Comment by rarkins** [comment, marked as answer]
Renovate should do that automatically using the format in your example

**Comment by SimonAdameit** [comment]
Okay, thanks! I thought it wouldn't, as I got `Dependency [...] has unsupported value prod` in the renovate log.

**Reply by rarkins** [reply]
That's a generic log message which essentially means "this doesn't look like a version". It should not stop the digest updates though
